src/uu/df/src/df.rs

```diff
@@ -448,7 +448,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
                 Err(error) => {
```

Comment:
I think this could be:
```rust
let filesystems = get_all_filesystems(&opt)
    .map_err_context(|| "cannot read table of mounted file systems".into())?;
```

See the [docs for `UIoError`](https://docs.rs/uucore/latest/uucore/error/struct.UIoError.html).

Sidenote: if you have any ideas about how to make this feature more discoverable/intuitive, I would be happy to hear it, because first-time contributors often miss this.

Reply:
I made the requested changes in the new commit, thanks for the suggestion.

It would be useful for first-time contributors to explicitly recommend this way of error construction in the documentation.